packages/myst-to-react/src/code.tsx

```diff
@@ -97,20 +97,23 @@ export function CodeBlock(props: Props) {
           <DocumentIcon
             width="16px"
             height="16px"
-            className="myst-code-filename-icon self-center flex-none inline-block text-gray-500 dark:text-gray-100"
+            className="inline-block flex-none self-center text-gray-500 myst-code-filename-icon dark:text-gray-100"
           />
-          <div className="myst-code-filename-title self-center p-2 text-sm leading-3 prose text-slate-600 dark:text-white">
+          <div className="self-center p-2 text-sm leading-3 myst-code-filename-title prose text-slate-600 dark:text-white">
             {filename}
           </div>
         </div>
       )}
-      <SyntaxHighlighter {...highlighterProps} className="myst-code-body block p-3 hljs">
+      <SyntaxHighlighter
+        {...highlighterProps}
+        className="block overflow-auto p-3 myst-code-body hljs"
+      >
```

Comment:
`overflow-auto` is the only actual change. Rest is just formatting from `headwind`...!